templates/tab_fact/tab_fact/templates.yaml

```diff
@@ -0,0 +1,128 @@
+dataset: tab_fact
+subset: tab_fact
+templates:
+  137a6f5d-fdcd-4849-ba3c-7ae572285ef9: !Template
+    id: 137a6f5d-fdcd-4849-ba3c-7ae572285ef9
+    jinja: '{% if label %}
+
+      Passage: "{{statement}}"
+
+
+      Table: "{{table_text}}"
+
+
+      Give a suitable caption for the table. Use the statement as a supporting doc.
+      |||
+
+      {{table_caption}}
+
+      {% endif %}'
+    name: tab_fact_7
+    reference: Generate Table Caption
+    task_template: false
+  1f0606bd-0453-427f-8cc5-ab996aff680e: !Template
+    id: 1f0606bd-0453-427f-8cc5-ab996aff680e
+    jinja: 'table ==> passage?
+
+
+      table:
+
+      "{{table_caption}}"
+
+
+      {{table_text}}
+
+
+      statement: "{{statement}}" |||
+
+
+      {{["No","Yes"][label]}}'
+    name: tab_fact_4
+    reference: 'Input: Table Caption, Table; Label: Yes/No (mathematical "implication"
+      symbol)'
+    task_template: true
+  33e3dbc2-3b1b-4891-8c78-2b575dd3ec35: !Template
+    id: 33e3dbc2-3b1b-4891-8c78-2b575dd3ec35
+    jinja: "Parse the following table:\n\nTable Caption: \"{{table_caption}}\"\n\n\
+      Table:\n\n{{table_text}}\n\nNote: {{\"#\"}} is the delimiter between columns;\
+      \ {{\"newline\"}} is the delimiter between rows.\n\nFrom the above table, the\
```

Comment:
`{{"newline"}} is the delimiter between rows` i don't see newline in the table....
<img width="1281" alt="Capture d’écran 2021-06-14 à 6 20 41 PM" src="https://user-images.githubusercontent.com/16107619/121966797-4177c700-cd3d-11eb-8487-35d98fa7d3db.png">

Reply:
Oh, that's strange. When I copy the "table_text", this is how it appears:
```"date#result#score#brazil scorers#competition\nmay 11 , 1919#w#6 - 0#friedenreich (3) , neco (2) , haroldo#south american championship\nmay 18 , 1919#w#3 - 1#heitor , amílcar , millon#south american championship\nmay 26 , 1919#d#2 - 2#neco (2)#south american championship\nmay 29 , 1919#w#1 - 0#friedenreich#south american championship\njune 1 , 1919#d#3 - 3#haroldo , arlindo (2)#taça roberto cherry\n"```

Another sample:
```"driver#race 1#race 2#race 3#points\nglenn seton#2#2#1#78\nalan jones#1#3#2#76\nmark skaife#3#1#3#74\nlarry perkins#5#4#4#67\nsteve reed#10#5#5#59\nbrad jones#7#7#6#58\nbob pearson#6#6#7#58\nterry finnigan#9#8#8#51\nkevin waldock#8#9#9#49\nkevin heffernan#13#10#10#41\nsteven ellery#14#11#11#37\nglenn mason#12#12#12#36\nphillip browne#15#13#13#30\njim richards#4#ret#ret#17\nneil schembri#11#ret#ret#10\n"```.

"\n" is present at the end of every row.

Reply:
I copied the above examples from the json in the sidebar

Reply:
oooh ok i understand, it's the rendering that removed the "\n"